tests/test_runtime/test_lr_updater.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmedit.core.scheduler.lr_updater import (LinearLrUpdaterHook,
+                                              ReduceLrUpdaterHook)
+
+
+class FakeLogBuffer(object):
+
+    def __init__(self) -> None:
+        self.output = {'PSNR': 30}
+
+
+class FakeRunner(object):
+
+    def __init__(self) -> None:
+        self.epoch = 0
+        self.max_epochs = 10
+        self.iter = 0
+        self.max_iters = 100
+        self.optimizer = 1
+        self.outputs = dict(log_vars=dict(loss=1))
+        self.eval_result = {'PSNR': 30}
+        self.log_buffer = FakeLogBuffer()
+
+
+def test_linear_lr_updater_hook():
+    fake_runner = FakeRunner()
+
+    lr_updater = LinearLrUpdaterHook(by_epoch=False)
+    lr_updater.get_lr(fake_runner, 1)
+
+    lr_updater = LinearLrUpdaterHook(by_epoch=True)
+    lr_updater.get_lr(fake_runner, 1)
+    lr_updater.start = 10
+    lr_updater.get_lr(fake_runner, 1)
```

Comment:
We need to at least assert something in the test, e.g. if the learning rate reduces to a desired value.
Same for `test_reduce_lr_updater_hook`

Reply:
Is this another feature irrelevant to reducing on the plateau?

Reply:
> Is this another feature irrelevant to reducing on the plateau?

Yes, it is another feature irrelevant to reducing on the plateau, which is in lack of unit test